SWLOR.Game.Server/Feature/LootTableDefinition/TatooineLootTableDefinition.cs

```diff
@@ -75,5 +87,15 @@ private void TuskenRaider()
                 .AddItem("map_038", 20, 1, true)
                 .AddItem("map_036", 1, 1, true);
         }
+
+        private void TuskenCrate()
```

Comment:
I tried to separate creature spawns out from resources/scavenge sites. There should be a TatooineResources class where the other resources are. Can you move this one to it instead?